e2echart/e2e-chart-template.html

```diff
@@ -313,6 +326,68 @@ <h5 class="modal-title">Resource</h5>
         return [item.locator, ` (${roles})`, m];
     }
 
+    function etcdLeadershipLogsValue(item) {
+        if (item.tempSource == "EtcdLeadership") {
+
+            // For leader-found, a 'term' is expected to be present; this line attempts to
+            // get the value of 'term' and if not there, we'll default to ''.
+            let term = item.tempStructuredMessage?.annotations?.['term'] ?? '';
+            if (term.length == 0) {
+                // For leader-lost, leader-elected, and leader-missing, term is in the humanMessage.
+                // Ensure it exists and if not, just default to 'noTerm'
+                term = item.tempStructuredMessage?.humanMessage ?? 'noTerm';
+            }
+            foundOrLost = item.tempStructuredMessage.reason
+
+            // If source is EtcdLeadership, then node is present.
+            nodeVal = item.tempStructuredLocator.keys['node']
+
+            let leaderColor ="EtcdOther"
+            switch (foundOrLost) {
+                case "leader-found":
+                    leaderColor = "EtcdLeaderFound";
+                    break;
+                case "leader-lost":
+                    leaderColor = "EtcdLeaderLost";
+                    break;
+                case "leader-elected":
+                    leaderColor = "EtcdLeaderElected";
+                    break;
+                case "leader-missing":
+                    leaderColor = "EtcdLeaderMissing";
+                    break;
+            }
+            return [`${nodeVal} term/${term}`, ` (${foundOrLost}=${nodeVal})`, leaderColor]
+        } else {
+
+            let nodeVal = 'noNode'
+            if (item.tempStructuredLocator.keys.hasOwnProperty('etcd-member') &&
+                item.tempStructuredLocator.keys.hasOwnProperty('node')) {
+                if (item.tempStructuredLocator.keys['node'].length > 0) {
+                    nodeVal = item.tempStructuredLocator.keys['node']
+                }
+            }
+
+            const etcdMemberVal = item.tempStructuredLocator.keys['etcd-member'] || ''
+            if (nodeVal.length == 0 && etcdMemberVal.length == 0) {
+                return [`empty`, `(empty)`, 'EtcdOther']
+            }
+
+            if (nodeVal.length > 0 && etcdMemberVal.length > 0) {
+                const term = item.tempStructuredMessage.annotations['term']
+                return [`Leader ${nodeVal} term/${term}`, ` etcd-member/${etcdMemberVal}`, 'EtcdLeading']
+            }
+
+            if (nodeVal == "noNode" && etcdMemberVal.length == 0) {
+                // If node is in the locator, use it.
+                nodeVal = item.tempStructuredLocator.keys['node'] || 'noNode'
+                return [`${nodeVal}`, ` (etcd-member=none)`, 'EtcdOther']
```

Comment:
When nodeVal is noNode, these intervals are pretty confusing. Is that showing any useful information at all?